app/bundles/CoreBundle/Helper/ExitCode.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\CoreBundle\Helper;
+
+/**
+ * Holds exit code constants for commands.
+ */
+final class ExitCode
+{
+    /**
+     * The command completed successfully.
+     */
+    public const SUCCESS = 0;
+
+    /**
+     * The command exited with some kind of error.
+     */
+    public const FAILURE = 1;
+
```

Comment:
This can be replaced with https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/Console/Command/Command.php#L38-L39 now.